portugues/spray/joaninha/comer/comer.md

```diff
@@ -1 +1,3 @@
-Hm... o sabor parece bem exótico!
\ No newline at end of file
+Hm... o sabor parece bem exótico!
+
+[Ficou com dor de barriga?](joaninha/comer/comer/dor-de-barriga.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.